docs/containers.rst

```diff
@@ -0,0 +1,53 @@
+Containers
+==========
+
+xonsh publishes a handful of containers, primarily targeting CI and automation use-cases. All of them are published on `Docker Hub <https://hub.docker.com/u/xonsh>`_.
```

Comment:
You must have double underscore after this, to make it an anonymous link. Otherwise docutils complains about "Duplicate explicit target name", because you use the same link name on line 25, 33, and 41.